portfolio/src/main/webapp/script.js

```diff
@@ -61,3 +61,9 @@ function makeLarger(pic){
   const popupDescrip = document.getElementById("popup-descrip");
   popupDescrip.innerHTML = pic.alt;
 }
+
+function getFromDataServlet(){
+    fetch('/data').then(response => response.text()).then((message) => {
+    document.getElementById('message-container').innerHTML = message;
```

Comment:
Please verify whether you wanted innerHTML or innerText.

- *innerHTML*: Honors any HTML tags in the string. If used on plain text, may interpret the text wrong if it contains certain symbols or sequences.
- "innerText": Treats the string as pure text. If it contains any HTML symbols or sequences, they will be encoded so that the user-visible result is the text of the string.

Reply:
Thank you for the clarification.  The message that is being sent from the server is "<h1>Hello Isabel!</h1>" so I believe innerHTML is correct.  In future steps, the message may not include tags so I will keep this in mind